src/main/features/offer/views/declaration.njk

```diff
@@ -45,35 +43,34 @@
         }}
       </p>
 
-      <p>{{ t('If you both keep to this agreement:') }}</p>
-      <ul class="list list-bullet">
-        <li>{{ t('the original claim and any counterclaims will be closed') }}</li>
-        <li>{{ t('neither of you can make any further claims relating to this case') }}</li>
-      </ul>
+      <p>{{t('This agreement puts the claim on hold. Neither party can make any further claims relating to this case, other than to enforce it.')}}</p>
+
+      <p>{{t('Either party can view and download this agreement from their money claims account. Both parties should keep a copy of this agreement.')}}</p>
 
-      <p>{{ t('This agreement is confidential until you’ve both signed it. After that it can be used in court, for example if the agreement is broken.') }}</p>
 
       <h3 class="heading-medium">{{ t('If the agreement is broken') }}</h3>
 
-      <p>{{ t('Either of you can apply to:') }}</p>
+      <p>{{ t('Either party can apply to:') }}</p>
       <ul class="list list-bullet">
         <li>{{ t('request a judgment for any money still owed from this agreement') }}</li>
-        <li>{{ t('restart the original claim or counterclaim') }}</li>
+        <li>{{ t('restart the original claim or counterclaim for the full amount of the claim') }}</li>
+        <li>{{ t('ask the court to enforce any of the steps to be taken')}}
       </ul>
 
       <p>
         {{
-          t('<a href="{{ n244formUri }}">Complete form N244</a> and send it to the <a href="{{ countyCourtBusinessCentreUri }}">County Court Business Centre</a>. You need to say on the form which of the 2 options you’re applying for.', {
+          t(' To apply, <a href="{{ n244formUri }}">complete form N244</a> and send it to the <a href="{{ countyCourtBusinessCentreUri }}">County Court Business Centre</a>. You need to say on the form which of the 3 options you’re applying for.', {
```

Comment:
can you remove the extra space at the start please?

Reply:
I've done that now - also fixed the code for the defendant name. Would be good to see this before we push live, maybe tomorrow ideally, I can't preview while editing in live due to memory issues which makes it a bit tricky, especially with a page like this which was quite different to the prototype.